test/oli_web/controllers/static_page_controller_test.exs

```diff
@@ -135,6 +135,18 @@ defmodule OliWeb.StaticPageControllerTest do
     end
   end
 
+  describe "student login" do
+    test "shows student login view", %{conn: conn} do
+      conn = get(conn, Routes.static_page_path(conn, :index))
+
+      assert response(conn, 200) =~
```

Comment:
This test strikes me as a bit too specific with respect to the exact html being checked, which could make it fragile to formatting or style tweaks. Can we limit this to only check for the element or text in question, e.g. "OLI Torus"?

Reply:
Of course. I change it!